lib/components/user/with-logged-in-user-support.js

```diff
@@ -37,26 +37,32 @@ export default function withLoggedInUserSupport (WrappedComponent, requireLogged
  * Upon completion (or if no user is logged in or if auth is disabled), it renders children.
  */
 class UserLoaderScreen extends Component {
-  componentDidUpdate () {
-    const { auth, fetchOrInitializeUser, loggedInUser } = this.props
+  async componentDidUpdate () {
+    const { auth0, fetchOrInitializeUser, loggedInUser } = this.props
 
-    // Once accessToken is available, proceed to fetch or initialize loggedInUser.
-    if (auth && auth.accessToken && !loggedInUser) {
-      fetchOrInitializeUser(auth)
+    if (auth0 && auth0.isAuthenticated && !loggedInUser) {
```

Comment:
Please use a comment to describe this condition that we're checking.

Reply:
This condition appears in multiple places, so replaced with a method (with jsdoc) in https://github.com/opentripplanner/otp-react-redux/pull/233/commits/0eedf9d0b11454855e64acc0c8c2569f5240b109